pyproject.toml

```diff
@@ -2,7 +2,7 @@
 # https://github.com/plone/meta/tree/main/config/default
 # See the inline comments on how to expand/tweak this configuration file
 [build-system]
-requires = ["setuptools>=68.2"]
+requires = ["setuptools>=68.2,<77"]
```

Comment:
What happened in setuptools 77? It might be good to add a comment with that information, with a link to an issue to update the pin when the dust settles.

Reply:
Also this needs to go into `meta.toml`, right?

Reply:
`setuptools` 77 breaks building some/most of our packages, because it changes how you should identify your license.  See https://github.com/plone/meta/issues/273.

The `<77` will be set by reapplying `plone.meta`, so there is no need to change anything in `meta.toml`.